new-evaluation.css

```diff
@@ -140,4 +144,15 @@
 			padding: 0px;
 			margin: 0px;
 			border: 2px 0px;
-		}
\ No newline at end of file
+		}
+
+		legend {
+  			margin: 10px 20px;
+  			
+  		}
+
+  		@media only screen and (max-device-width: 480px) {
+			.evaluation_form_submit {text-transform: uppercase; }
```

Comment:
I used this media query cause Alex asked me if I was able to make the submit button be displayed in capital case and in uppercase only on mobile devices.